examples/arrays/arrays.bal

```diff
@@ -22,25 +23,27 @@ public function main() {
 
     // Arrays are an iterable type and arrays support functional iteration operations such as `.map()`, `.filter()`, and
     // `.reduce()`.
-    int[] doubled = b.map(function (int value) returns int {
-            return value * 2;
-        });
+    int[] doubled = b.map(function(int value) returns int {
+                              return value * 2;
+                          });
     io:println("Doubled: ", doubled);
 
     // An array can be sorted using `.sort()`. The `array:DESCENDING` is the `direction` in which sort should be done.
     // The default sort direction is `ASCENDING`. The `isolated function` returns a `string[]` for each
     // member, which is used as a `key` to sort the members. If the member type of the array
     // is not sorted, then the `key` function must be specified.
     // It returns an `array` consisting of the members of `b` in the sorted order.
-    int[] sortedArray = b.sort(array:DESCENDING, isolated function (int value)
-    returns string[] {
-            if (value < 5) {
-                return ["A",value.toString()];
-            } else {
-                return ["B",value.toString()];
-            }
-        });
-    io:println("Sorted Array: ", sortedArray);
+    int[] sortedArray = b.sort();
+    io:println("Array sorted in ascending order: ", sortedArray);
+
+    sortedArray = b.sort(array:DESCENDING, isolated function(int value) returns string[] {
```

Comment:
Isn't the character count > 80 here? 

https://github.com/ballerina-platform/ballerina-distribution/blob/master/doc-guidelines.md#ballerina-by-examples-guidelines

Reply:
Fixed it by reverting to previous formatting.
3391dd21aeb9e78228947d40e8436a90f226304a